app/static/css/spinner.css

```diff
@@ -0,0 +1,18 @@
+.spinner {
+  margin: auto;
+  border: 10px solid #e6d4c8;
+  border-top: 10px solid #664bd2;
+  border-radius: 50%;
+  width: 60px;
+  height: 60px;
+  animation: spin 1.5s linear infinite;
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}

```

Comment:
I'd rather hold off on a separate CSS file until we're using the rules in more than one place.